backend/src/Github/Service/GithubPRService.php

```diff
@@ -86,11 +86,11 @@ public function createConnectionPRTask(ProjectAgentConnection $connection, int $
         [$owner, $repo] = explode('/', $connection->getConfigValue('repository'));
         $task = new Task();
         $task->name = "{$repo} #{$githubPRId}: {$prDetails->title}";
-        $task->description = $prDetails->body . "\nCommits:\n {$commitNames}";
+        $task->description = trim($prDetails->body . "\nCommits:\n {$commitNames}");
         $task->status = TaskStatus::NEW;
         $task->source = 'github-pr';
```

Comment:
Very good catch on correcting the PR URL construction by adding `$owner` to the URL string. Previously, the URL was potentially malformed without the repository owner.